examples/pubsub/tutorial_pubsub_mqtt_publish.c

```diff
@@ -40,21 +40,24 @@
  * options is provided, the system's default certificate location will be used.
  */
 
-#include "open62541/server.h"
-#include "open62541/server_config_default.h"
+#include <open62541/plugin/log_stdout.h>
+#include <open62541/server.h>
+#include <open62541/server_config_default.h>
+
 #include "ua_pubsub.h"
-#include "ua_network_pubsub_mqtt.h"
-#include "open62541/plugin/log_stdout.h"
+
 #include <signal.h>
 
+#include <open62541/plugin/pubsub_mqtt.h>
+
 #define CONNECTION_NAME              "MQTT Publisher Connection"
 #define TRANSPORT_PROFILE_URI        "http://opcfoundation.org/UA-Profile/Transport/pubsub-mqtt"
 #define MQTT_CLIENT_ID               "TESTCLIENTPUBSUBMQTT"
 #define CONNECTIONOPTION_NAME        "mqttClientId"
 #define PUBLISHER_TOPIC              "customTopic"
 #define PUBLISHER_METADATAQUEUENAME  "MetaDataTopic"
 #define PUBLISHER_METADATAUPDATETIME 0
-#define BROKER_ADDRESS_URL           "opc.mqtt://127.0.0.1:1883"
+#define BROKER_ADDRESS_URL           "opc.mqtt://test.mosquitto.org:8883"
```

Comment:
isn't `#define BROKER_ADDRESS_URL           "opc.mqtt://127.0.0.1:1883"`  correct to make sure automated tests work?